homework.md

```diff
@@ -4,110 +4,108 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website look better (style, size, position, color and so on)
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
-
-}
+'''css
+#h1 {}
```

Comment:
Hi Irina, work looks great! Just a quick note, in order to apply the css to all h1 tags, the hash reference (#) needs to be removed. Otherwise the styling would register h1 as an id attribute belonging to a tag.

Reply:
Thank you, Hamra